### DISCUSSION TITLE
[Bug/Feature] Back to list, remember scroll position

### DISCUSSION THREAD

**Discussion by fakethinkpad85** [mealie-recipes/mealie#2609] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/hay-kot/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

When going back to the list/cookbook page from a recipe the scroll position is not restored but need to remember where i was and scroll back each time i look at a recipe and go back to the list. Using a phone this becomes annoying very fast. 

### Please provide a concise description of the feature that would resolve your issue.

 Return to the last scroll position

### Please consider and list out some caveats or tradeoffs made in your design decision

 Development time.

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by solluh** [comment]
Is this request somehow in progress? My wife is pretty annoyed that she has to keep scrolling down every time she leaves a recipe page.
Thank you!

**Reply by boc-the-git** [reply]
No, I don't know of anyone working on this.
Seems a reasonable enough request; PRs are welcome/appreciated!

**Comment by fakethinkpad85** [comment]
Bump, surely there must be alot of users on here having the same iritation? still cant get family and friends to use it due to this reason.

**Comment by solluh** [comment]
At least, this suggestion is implemented today.
Thanks!
